### ISSUE TITLE
[Harbor] In Harbor when you go to `about` it shows version as empty.

### ISSUE THREAD

**Issue by paulczar** [bitnami/charts#5192]
I guess this is more of an image build problem, but when I deploy harbor via the chart, the about->version in the web app shows as empty.

**Comment by miguelaeh** [comment]
Hi @paulczar ,
I can reproduce it:
<img width="303" alt="image" src="https://user-images.githubusercontent.com/36923089/105687370-a9498480-5ef8-11eb-8b3f-f622fd781a01.png">
I will open an internal task to check this when we have the chance. However, I can't provide an ETA. 
Thank you very much!

**Comment by miguelaeh** [comment]
Hi guys!
Sorry IT was closed because I forgot to add the `on-hold` label

**Comment by FraPazGal** [comment]
Hello @paulczar, @holzman,

This issue is now fixed in the latest chart's major, `12.0.0`. You can check it so by doing:

```
$ helm repo update
$ helm search repo bitnami/harbor
NAME          	CHART VERSION	APP VERSION	DESCRIPTION
bitnami/harbor	12.0.0       	2.4.1      	Harbor is an open source trusted cloud-native r...
$ helm install harbor bitnami/harbor
```

<img width="246" alt="Screenshot 2022-03-09 at 13 43 32" src="https://user-images.githubusercontent.com/26460210/157444085-5bc7ca64-f912-4ff2-9e40-449ed6ee4add.png">

Thanks for reporting this bug and please do not doubt to reach out to us if there is any further issue.
